wasm/src/wasmedge.rs

```diff
@@ -263,9 +276,7 @@ impl ProcessLifecycle<ExecProcess> for WasmEdgeExecLifecycle {
     }
 
     async fn delete(&self, _p: &mut ExecProcess) -> containerd_shim::Result<()> {
-        Err(Error::Unimplemented(
```

Comment:
Exec in wasm container is still not supported, so should return `Err` here.

Reply:
done